src/Phinx/Db/Adapter/SQLiteAdapter.php

```diff
@@ -784,70 +804,67 @@ protected function getDropIndexByNameInstructions($tableName, $indexName)
      */
     public function hasPrimaryKey($tableName, $columns, $constraint = null)
     {
-        $primaryKey = $this->getPrimaryKey($tableName);
-
-        if (empty($primaryKey)) {
-            return false;
+        if (!is_null($constraint)) {
+            throw new \InvalidArgumentException('SQLite does not support named constraints.');
         }
 
-        $missingColumns = array_diff((array)$columns, (array)$primaryKey);
+        $columns = array_map('strtolower', (array)$columns);
+        $primaryKey = array_map('strtolower', $this->getPrimaryKey($tableName));
 
-        return empty($missingColumns);
+        if (array_diff($primaryKey, $columns)) {
+            return false;
+        } elseif (array_diff($columns, $primaryKey)) {
+            return false;
+        } else {
+            return true;
+        }
```

Comment:
```php
if (array_diff($primaryKey, $columns)) {
    return false;
}
if (array_diff($columns, $primaryKey)) {
    return false;
}
return true;
```
please

Reply:
I removed the else, and also did an or comparison as in the other case.